app/controllers/appointments_controller.rb

```diff
@@ -2,22 +2,25 @@ class AppointmentsController < AdminController
   include FacilityFiltering
   include Pagination
 
+  before_action :set_patient_search_filters, only: [:index]
   before_action :set_appointment, only: [:update]
 
   def index
     authorize [:overdue_list, Appointment], :index?
 
-    @patient_summaries = policy_scope([:overdue_list, PatientSummary])
-                           .overdue
-                           .order(risk_level: :desc, next_appointment_scheduled_date: :desc)
+    @search_filters = index_params[:search_filters] || []
+
+    @patient_summaries = apply_search_filters
 
     if current_facility
       @patient_summaries = @patient_summaries.where(next_appointment_facility_id: current_facility.id)
     end
+    @patient_summaries = @patient_summaries.order(risk_level: :desc, next_appointment_scheduled_date: :desc)
 
     respond_to do |format|
       format.html do
         @patient_summaries = paginate(@patient_summaries).includes(:next_appointment, patient: :appointments)
+        render layout: "overdue"
```

Comment:
Need a new layout for the new sidebar.

Reply:
> Does the existing layout introduce a bunch of grid things that make it hard to incorporate the new design?

The existing default layout doesn't really have any notion of a sidebar at all, so it would take an overhaul if we wanted to build in sidebar ability.

> PS. can we re-use whatever layout the My Facilities pages are using?

That is basically what this is, except copy / pasted =).  My Facilities had a bunch of code that was facilities specific, so we couldn't direclty reuse.  If all dashboards will all have some sort of sidebar in the future (/cc @dburka ?), then we could do a subsequent PR to get a common layout for dashboards that supports a sidebar in a cleaner way.

Reply:
I started a broader convo in slack about this https://simpledotorg.slack.com/archives/CK95QFJA2/p1588698688024800.

Reply:
I'll let you plant the results of that convo into this PR 👍